arcjet-bun/index.ts

```diff
@@ -144,12 +145,19 @@ function toArcjetRequest<Props extends PlainObject>(
   const headers = new ArcjetHeaders(request.headers);
 
   const url = new URL(request.url);
-  const ip = findIP(
+  let ip = findIP(
     {
       ip: ipCache.get(request),
     },
     headers,
   );
+  if (ip === "") {
+    // If the `ip` is empty but we're in development mode, we default the IP
+    // so the request doesn't fail.
+    if (env.NODE_ENV === "development" || env.ARCJET_ENV === "development") {
+      ip = "127.0.0.1";
```

Comment:
Do you think it's worth logging that we're doing this? Keeps the developer informed about what's happening

Reply:
Yes, but I have to do that in a follow-up PR because the logger construction is being moved from the `arcjet` core package into each adapter (because log level needs to be detected from the env). I'll add TODOs that will be cleaned up by that PR.

Reply:
Logged https://github.com/arcjet/arcjet-js/issues/987